admin/metabox/class-metabox.php

```diff
@@ -420,78 +416,12 @@ private function get_advanced_meta_section() {
 	 * @return string
 	 */
 	private function get_buy_premium_link() {
-		return sprintf( "<div class='%s'><a href='#wpseo-meta-section-premium' class='wpseo-meta-section-link'><span class='dashicons dashicons-star-filled wpseo-buy-premium'></span>%s</a></div>",
+		return sprintf( "<div class='%s'><a target='_blank' rel='noreferrer noopener' href='https://yoa.st/add-keywords-popup' class='wpseo-meta-section-link'><span class='dashicons dashicons-star-filled wpseo-buy-premium'></span>%s</a></div>",
```

Comment:
Please use the `WPSEO_Shortlinker::get()` and the `esc_url` methods to wrap the shortlink.

While at this, can you convert the `"` and `'` in this piece. I really prefer to have double-quotes in HTML attributes.